l2geth/miner/worker.go

```diff
@@ -771,6 +779,7 @@ func (w *worker) commitTransaction(tx *types.Transaction, coinbase common.Addres
 	}
 	snap := w.current.state.Snapshot()
 
+	start := time.Now()
 	receipt, err := core.ApplyTransaction(w.chainConfig, w.chain, &coinbase, w.current.gasPool, w.current.state, w.current.header, tx, &w.current.header.GasUsed, *w.chain.GetVMConfig())
```

Comment:
Do we also want to collect metrics in the case where it reverts and returns early?

Reply:
Reverts aren't errors. The sort of errors that happen here are invalid transaction fields, bad nonce, etc, which aren't useful for our metrics.